libraries/botbuilder-dialogs/tests/skillDialog.test.js

```diff
@@ -91,6 +91,8 @@ describe('SkillDialog', function() {
                 activityToSend.text = activityToSendText;
                 const client = new DialogTestClient(Channels.Test, dialog, { activity: activityToSend }, undefined, conversationState);
 
+                strictEqual(0, dialogOptions.conversationIdFactory.createCount);
```

Comment:
```suggestion
                strictEqual(dialogOptions.conversationIdFactory.createCount, 0);
```

The value being tested is the first value in [`assert.strictEqual()`](https://nodejs.org/api/assert.html#assert_assert_strictequal_actual_expected_message)